### DISCUSSION TITLE
Welcome to Prism-OS Discussions!

### DISCUSSION THREAD

**Discussion by terminal-cs** [Project-Prism/Prism-OS#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * be respectful to other users
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by theopcoder** [comment]
Hello, is there an ISO for this OS?

**Reply by terminal-cs** [reply]
Yes, you can find the latest stable release in the releases tab. It will not work on VMware anymore but qemu and real hardware should work. If you want to get the true latest version of Prism, you can just compile the source code yourself (stability not guaranteed.)

**Reply by theopcoder** [reply]
Alr, ty!
ps, for the website merge, i forgot to mention that i wasn't 100% who the owner/main leader of this repo is so I just put <put author here>. Feel free to change that at any time!

**Reply by theopcoder** [reply]
No problem! Hopefully this becomes the next big OS!

**Reply by terminal-cs** [reply]
> No problem! Hopefully this becomes the next big OS!

We plan on it

**Comment by theopcoder** [comment]
Is there a Discord for the project?

**Reply by terminal-cs** [reply]
> we only have a discord group chat for the time being

🤔Maybe we could make one and move out conversation to a admin only chanel, and people can still ask in other channels

**Comment by theopcoder** [comment]
Is the project called PrismOS all in one word or Prism OS, two words?

**Reply by terminal-cs** [reply]
It is called Prism OS, you will only see it as one word where you can't use spaces.

**Reply by theopcoder** [reply]
alright!
